modules/Formal Assessment/internalAssessment_manage_addProcess.php

```diff
@@ -22,7 +22,8 @@
 include '../../gibbon.php';
 
 $gibbonCourseClassID = $_GET['gibbonCourseClassID'] ?? '';
-$URL = $session->get('absoluteURL').'/index.php?q=/modules/'.getModuleName($_GET['address'])."/internalAssessment_manage_add.php&gibbonCourseClassID=$gibbonCourseClassID";
+$address = $_POST['address'] ?? '';
+$URL = $session->get('absoluteURL').'/index.php?q=/modules/'.getModuleName($address)."/internalAssessment_manage_add.php&gibbonCourseClassID=$gibbonCourseClassID";
```

Comment:
The original code uses a $_GET['address'] and the variable has changed this to a $_POST['address'], which although may have not resulted in an error (as many forms do post an address variable), it's best to retain the original logic in this type of refactoring.